solana-programs/audius_eth_registry/tests/tests.rs

```diff
@@ -623,7 +623,7 @@ async fn validate_signature() {
 }
 
 #[tokio::test]
-async fn validate_signature_with_wrong_data() {
+async fn validate_signature_with_wrong_signer() {
```

Comment:
the reason 'wrong' data is no longer necessary is because we don't actually need to verify the signature itself - on the left side we were passing an empty signature object but now we do not need to since the verification is redundant